app/views/users/_foster_filters.html.erb

```diff
@@ -0,0 +1,56 @@
+<h4>Foster Filters</h4>
+
+<form class=form-inline">
+  <input type="hidden" name="foster" value="true"/>
+
+  <label class="pull-right">
+    Available
+    <%= select_tag "available_to_foster", options_for_select(@options, params[:available_to_foster]), class: 'input-mini' %>
+  </label>
+
+  <label class="pull-right">
+    Waiver
+    <%= select_tag "available_to_foster", options_for_select(@options, params[:available_to_foster]), class: 'input-mini' %>
```

Comment:
The waiver is an Attachment object to User.  Basically an uploaded pdf that they signed.